Automattic/wp-calypso
# Global sidebar - update alternative text color when selected

Related to # p1712077790192889-slack-C03NLNTPZ2T

## Proposed Changes

* defines a variable for alternative text color when selected, sets this as `studio-gray-10`
* uses this variable for the alternative text used in p2 section of the reader

BEFORE
<img width="301" alt="Screenshot 2024-04-02 at 1 38 19 PM" src="https://github.com/Automattic/wp-calypso/assets/28742426/14e7df8d-c7fb-447b-a290-73f250ec7849">

AFTER
<img width="285" alt="Screenshot 2024-04-02 at 1 38 24 PM" src="https://github.com/Automattic/wp-calypso/assets/28742426/1477c8fb-199a-44a6-8b5d-f8e476ddf05c">




## Testing Instructions



* Run this branch of calypso.
* select a p2 in the reader
* verify the "X minutes ago" text color is updated and clearly visible.

## Pre-merge Checklist
